src/tests/JIT/IL_Conformance/Old/Conformance_Base/ldarg_i.il

```diff
@@ -24,7 +24,6 @@
 	ldsflda		int32 _ldarg::FOO
 	ldc.i4		0x2
 	add
-	conv.i4
 	ldsflda		int32 _ldarg::FOO		
 	call	int32 _ldarg::args(int32)
```

Comment:
I'm confused: the main branch contains:
```
call	native int _ldarg::args(native int)
```
but GitHub is showing me the baseline of:
```
call	int32 _ldarg::args(int32)
```
?